lib/type.nunjucks

```diff
@@ -0,0 +1,44 @@
+{% if typeName == 'object' %}
+  {{ typeName }}
+{% else %}
+  {% set typeOptions = typeWithName(typeName) %}
+  {% if typeOptions %}
+    {% set type = typeOptions.type %}
+
+    <div class='list-group-item'>
+
+      <p>{{ type.displayName }}</p>
+
+      {% if type.description %}
+        <div class='alert alert-info'>
+          {{ type.description }}
+        </div>
+      {% endif %}
+
+      <p>
+        <strong>Type:</strong>
+        {% for name in type.type %}
+          {{ name }}
+          {{ "" if loop.last else "," }}
+        {% endfor %}
+      </p>
+
+      {% if typeOptions.rootType == 'object' %}
+        <strong>Properties</strong>
+        <ul>
+          {% for key, item in propertiesOfType(type) %}
+            {% include "./item.nunjucks" %}
+          {% endfor %}
+        </ul>
+      {% endif %}
+
+      {% if type.example %}
+        <p><strong>Example</strong>:</p>
+        <pre><code>{{ type.example | escape }}</code></pre>
```

Comment:
We might need to be careful here since an example can be expressed as XML, JSON, and YAML. The first two can be handled with what you have done, but YAML is not represented as `string`. So it would probably not work.